lib/URI.pm

```diff
@@ -1085,6 +1116,34 @@ examples:
 This value can be set to ";" to have the query form C<key=value> pairs
 delimited by ";" instead of "&" which is the default.
 
+=back
+
+=head1 ENVIRONMENT VARIABLES
+
+=item URI_HAS_RESERVED_SQUARE_BRACKETS
```

Comment:
You need to insert an `=over` here to restart the list, please. That's why the test on the runner failed.